lib/rules/hidden-content.json

```diff
@@ -0,0 +1,14 @@
+{
+  "id": "hidden-content",
+  "matches": "hidden-content.js",
+  "tags": [],
```

Comment:
Let's tag this as experimental. @WilcoFiers @dylanb should we also tag it as a best-practice?